tests/get_link_stats_vals.py

```diff
@@ -0,0 +1,38 @@
+"""comparison for unit tests"""
```

Comment:
Oh! Can we not do this? It's much cleaner to keep test data out of source. Just create a json resource file for this.